packages/babel-helpers/src/helpers/createClass.ts

```diff
@@ -1,7 +1,13 @@
 /* @minVersion 7.0.0-beta.0 */
 
-import toPropertyKey from "toPropertyKey";
-function _defineProperties(target, props) {
+// @ts-expect-error Migrate in another PR
+import toPropertyKey from "toPropertyKey.ts";
```

Comment:
```suggestion
import toPropertyKey from "./toPropertyKey.ts";
```

(and it's already written in TS :) )